user_guide_src/source/libraries/encryption.rst

```diff
@@ -61,61 +62,60 @@ You don't need to worry about it.
 
 .. _configuration:
 
-Configuring the library
+Configuring the Library
 =======================
 
-The example above uses the default configuration settings,
-found in ``application/config/Encryption.php``.
+The example above uses the configuration settings found in ``app/Config/Encryption.php``.
 
 There are only two settings:
 
 ======== ===============================================
 Option   Possible values (default in parentheses)
 ======== ===============================================
-driver   Preferred handler (OpenSSL)
 key      Encryption key starter
+driver   Preferred handler (OpenSSL)
 ======== ===============================================
 
-You can over-ride any of these settings by passing your own ``Config`` object
-to the Services::
+You can replace the config file's settings by passing a configuration 
+object of your own to the ``Services`` call. The ``$config`` variable must me 
```

Comment:
should be `must be`, not `must me`, I'm pretty sure :)

Reply:
Picky, picky, picky. :)

Reply:
I know, right!? :)